test/DOM-updates-test.js

```diff
@@ -1,13 +1,17 @@
 const chai = require('chai');
 const expect = chai.expect;
 const spies = require('chai-spies');
-import domUpdates from './DOM-updates.js';
+import domUpdates from '../src/DOM-updates.js';
+
 
 chai.use(spies);
 
 describe('domUpdates', () => {
     beforeEach(() => {
         chai.spy.on(domUpdates, [
+            'populateCards',
+            'greetUser',
+            'tellMeYourMoneys'    
```

Comment:
Hilarious. But in all reality, maybe I should change this name. It's not unprofessional, just a bit goofy.